images/win/scripts/Installers/Install-Msys2.ps1

```diff
@@ -53,40 +53,38 @@ pacman.exe -Syuu  --noconfirm
 taskkill /f /fi "MODULES eq msys-2.0.dll"
 
 Write-Host "`n$dash Install msys2 packages"
-pacman.exe -S --noconfirm --needed --noprogressbar base-devel compression
+$msysPackages = (Get-ToolsetContent).MsysPackages.msys2
```

Comment:
nitpick:  create a new variable for `(Get-ToolsetContent).MsysPackages`

Reply:
Good idea, thanks!